x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -7,6 +7,7 @@
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { idx } from '@kbn/elastic-idx';
+import { identity } from 'lodash';
```

Comment:
I've noticed that we generally avoid new imports from lodash if we don't already use them in other places so that we maintain a smaller bundle size. i know `identity` does add much to the bundle, but you could avoid this new import by using `.filter(Boolean)` to achieve the same result.

Reply:
@ogupte You're right, I mistakenly assumed we were already using it. Replaced 👍